test-utils/state-viewer/src/commands.rs

```diff
@@ -64,14 +76,34 @@ pub(crate) fn dump_state(
         load_trie_stop_at_height(store, home_dir, &near_config, mode);
     let height = header.height();
     let home_dir = PathBuf::from(&home_dir);
-    let output_dir = home_dir.join("output");
 
-    let records_path = output_dir.join("records.json");
-    let new_near_config =
-        state_dump(runtime, state_roots.clone(), header, &near_config, &records_path);
-
-    println!("Saving state at {:?} @ {} into {}", state_roots, height, output_dir.display(),);
-    new_near_config.save_to_dir(&output_dir);
+    match dump_mode {
+        DumpStateMode::InMemory => {
+            let new_near_config =
+                state_dump(runtime, state_roots.clone(), header, &near_config, None);
+            let output_file = home_dir.join("output.json");
+            println!(
```

Comment:
It may be a good idea to use tracing here:
```
use tracing::info;

info!( message =  "Saving state at,
                ?state_roots,
                height,
                output_file.display(),
            );
```
This will print a nicely formatted output.

Reply:
```
@@ -82,11 +84,11 @@ pub(crate) fn dump_state(
             let new_near_config =
                 state_dump(runtime, state_roots.clone(), header, &near_config, None);
             let output_file = home_dir.join("output.json");
-            println!(
-                "Saving state at {:?} @ {} into {}",
-                state_roots,
+            info!(
+                message = "Saving state",
+                ?state_roots,
                 height,
-                output_file.display(),
+                output_file = ?output_file.display()
             );
             new_near_config.genesis.to_file(&output_file);
         }
```

Reply:
The use of `tracing` was discussed in one of the previous PRs: https://github.com/near/nearcore/pull/4867#discussion_r717534200

Reply:
@nikurt Yes, I didn't mean in this pr. It makes sense to be consistent. 

I didn't mean to use tracing now, but rather we should consider changing  to using `tracing` in general rather than `println`. That is out of scope of this PR.